objectRecognitionSource/src/test/resources/objRecImpl.json

```diff
@@ -2,5 +2,9 @@
    "name" : "OBJECT_RECOGNITION",
    "baseType" : "EXTENSION",
    "verticle" : "service:extensionSource",
-   "config" : {}
+   "config" : {},
+   "baseConfigProperties": [
+      "objRecConfig.dataSource.type",
+      "objRecConfig.neuralNet.type"
```

Comment:
wouldn't we typically want the camera address here, too? Or is that too tricky since it's dependent upon the datasource type?

Reply:
Yep- because different dataSource types offer different config options, I wasn't sure if we should add the `camera` or not. Now that I think of it, I'm pretty sure all of them require a `camera`, so I think I can add that without issue.